src/app/components/article/Articles.js

```diff
@@ -263,12 +271,14 @@ const ArticlesComponent = ({
             onChange={handleChangeSort}
           />
           <ArticleFilters
-            articleTypeReadOnly={Boolean(defaultFilters.article_type)}
+            articleTypeReadOnly={Boolean(defaultFilters.article_type && !savedSearch)}
```

Comment:
FYI: This `Boolean` became useless/redundant now that the prop value is a boolean expression.

Reply:
Done